lib/matplotlib/backend_bases.py

```diff
@@ -2500,6 +2501,9 @@ def key_press_handler(event, canvas, toolbar=None):
     if event.key in quit_keys:
         Gcf.destroy_fig(canvas.figure)
 
+    if event.key in quit_all_keys:
```

Comment:
I think we are going to leave it only inside `toolmanager`, please remove it from `backend_bases`